ddtrace/appsec/iast/_util.py

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env python3
-from functools import cache
 import sys
 
 from ddtrace.internal.logger import get_logger
 from ddtrace.settings import _config as config
+from ddtrace.vendor.psutil._compat import lru_cache
```

Comment:
We have the `callonce` helper in our internal `cache` module that might be better suited for the job here

Reply:
I tried our `callonce` from our internal `cache` but I removed it because it logs some deprecation warnings and that produces some test failures as the tests expect stdout and stderr to be empty.